docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
Yeah, the other thing that's kinda annoying about using the apt repository is that we can't pin a version easily (unless their apt repository is setup differently than most), so all versions of Zulip in production will suddenly stop installing if they ever ship a broken a version.

I think if we can get a pinned version of yarn to be installed into `zulip-npm-cache` (or its own `zulip-yarn-cache`, if we think that's better), that would be great.  I think calling that installation code from `install-node` would be cleaner (just in terms of the logic: `install-node` gets us our JS toolchain, and then `node_cache.py` gets the `node_modules` from there...), but could go the other way if there are compelling technical reasons...